src/www/widgets/widgets/carp_status.widget.php

```diff
@@ -74,14 +74,14 @@
 }
 if ($ipaddress) {
 ?> &nbsp;
-        <?=htmlspecialchars($status);?> &nbsp;
+        <?=htmlspecialchars(gettext($status));?> &nbsp;
```

Comment:
there are three values possoble if you look at the file MASTER, BACKUP, INIT and DISABLED - I will solve that with a switch-case